packages/app/app/components/PlayQueue/QueueBottomBar/index.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './styles.scss';
+import { Icon } from 'semantic-ui-react';
+import _ from 'lodash';
+
+export const QueueBottomBar= ({
+  addToDownloads,
+  items,
+  currentSong
+}) => {
+
+  const handleDownloads = () => {
+    if ((currentSong !== null && currentSong >= 0) && (items !== null && items !== undefined && items.length > 0)) {
+      addToDownloads(_.get(items, currentSong));
+    }
+  };
+
+  const bottomBarOptions = [{
+    link: '#',
+    iconName: 'triangle left',
+    onClickHandler: null
+  },
+  {
+    link: '#',
+    iconName: 'star',
+    onClickHandler: null
+  }, 
+  {
+    link: '#',
+    iconName: 'download',
+    onClickHandler: handleDownloads
+  }];
+  
+  return (
+    <div className={styles.queue_bottom_bar}>
+      Current Track
```

Comment:
Please don't add bare strings and use the `useTranslation` hook.
You can see how it's used in other components.
You will need to use `t('header-track')`, since this string is already localized.

Reply:
Changes done.

![image](https://user-images.githubusercontent.com/37588410/117738802-caf32100-b1f4-11eb-8f92-8a1303dfed2c.png)

![image](https://user-images.githubusercontent.com/37588410/117738826-d3e3f280-b1f4-11eb-9664-186afa8533e4.png)